packages/editor/src/client/routes/articleEditor.tsx

```diff
@@ -319,7 +320,15 @@ export function ArticleEditor({id}: ArticleEditorProps) {
     }
   }, [isNotFound])
 
-  const totalCharCount = countRichTextBlocksChars(blocks);
+  let totalCharCount = 0;
```

Comment:
you don't need this. you can set the count directly in the `useEffect` and don't need to save it first in the variable.